src/Faker/Provider/kk_KZ/Person.php

```diff
@@ -175,26 +175,21 @@ class Person extends \Faker\Provider\Person
         'Ісмет',
     ];
 
-    /**
-     * @param int $year
-     *
-     * @return int|null
-     */
-    private static function getCenturyByYear($year)
+    private static function getCenturyByYear(int $year): int
```

Comment:
This is a BC and not allowed

Reply:
Please clarify, that is private method, so it cannot affect any descendant classes

Reply:
Ah sorry you are ofc right